pages.de/common/pip.md

```diff
@@ -0,0 +1,25 @@
+# pip
+
+> Python package manager.
+> Einige Unterbefehle wie `pip install` sind separat dokumentiert.
+> Weitere Informationen: <https://pip.pypa.io>.
+
+- Installiere ein Paket (siehe `pip install` für weitere Beispiele):
+
+`pip install {{paketname}}`
```

Comment:
```suggestion
`pip install {{paket_name}}`
```

Reply:
Hi Navarroaxel, in german it's `Paketname`, do you still want me to split it?

Reply:
Let's keep it the same as in `pip-install` (@marchersimon approved it there)